### ISSUE TITLE
React native  runModelOnImage have an error "Cannot convert between a TensorFlowLite tensor with type UINT8 and a Java object of type [[[[F (which is compatible with the TensorFlowLite type FLOAT32)"

### ISSUE THREAD

**Issue by Duynguyen0897** [tensorflow/tensorflow#49853]
When I training data  and export to Tflite on centos7 using Anaconda3 with Tensorflow, it have warning
![image](https://user-images.githubusercontent.com/44739285/119945247-63dcb700-bfbf-11eb-8730-2d50d6bedf12.png)
The [model.tflite](https://drive.google.com/file/d/1L5KFlHSv1NajUM5Ltpp354O-i8AzKAkH/view?usp=sharing) and [labels.txt](https://drive.google.com/file/d/1eXDEO81Ach45qJTryBWX_O0pupS61GiW/view?usp=sharing) after export.
Below is my code and the datasets [here](https://drive.google.com/file/d/1SE8DJNU8ctaim2BptUMWZ3dkxPPVSpfT/view?usp=sharing).
```python
import os
import numpy as np
import tensorflow as tf
assert tf.__version__.startswith('2')

from tflite_model_maker import configs
from tflite_model_maker import ExportFormat
from tflite_model_maker import image_classifier
from tflite_model_maker import model_spec
from tflite_model_maker import ImageClassifierDataLoader
import matplotlib.pyplot as plt

data = ImageClassifierDataLoader.from_folder('images')
train_data, rest_data = data.split(0.8)
validation_data, test_data = rest_data.split(0.5)
plt.figure(figsize=(10,10))
for i, (image, label) in enumerate(data.gen_dataset().unbatch().take(25)):
  plt.subplot(5,5,i+1)
  plt.xticks([])
  plt.yticks([])
  plt.grid(False)
  plt.imshow(image.numpy(), cmap=plt.cm.gray)
  plt.xlabel(data.index_to_label[label.numpy()])
plt.show()

model = image_classifier.create(train_data, validation_data=validation_data)
model.summary()
loss, accuracy = model.evaluate(test_data)
def get_label_color(val1, val2):
  if val1 == val2:
    return 'black'
  else:
    return 'red'
plt.figure(figsize=(20, 20))
predicts = model.predict_top_k(test_data)
for i, (image, label) in enumerate(test_data.gen_dataset().unbatch().take(100)):
  ax = plt.subplot(10, 10, i+1)
  plt.xticks([])
  plt.yticks([])
  plt.grid(False)
  plt.imshow(image.numpy(), cmap=plt.cm.gray)

  predict_label = predicts[i][0][0]
  color = get_label_color(predict_label,
                          test_data.index_to_label[label.numpy()])
  ax.xaxis.label.set_color(color)
  plt.xlabel('Predicted: %s' % predict_label)
plt.show()
model.export(export_dir='.')
model.export(export_dir='./duy_tree', export_format=ExportFormat.LABEL)
```

Same code above but running on window, after export tflite model can be used.
Help me, please!

**Comment by tilakrayal** [comment]
@Duynguyen0897 ,

Can you please confirm the tensorflow version trying to execute the code.Thanks!

**Comment by Duynguyen0897** [comment]
@tilakrayal

The version I'm using is 2.3
But Same code and version above but running on window, after export tflite model can be used.

**Comment by pjpratik** [comment]
Hi @Duynguyen0897 

I have tried to reproduce the issue on Ubuntu 20.04 and did not get any mentioned warnings while conversion.

Please find the gist [here](https://colab.research.google.com/gist/pjpratik/69e864a99e7ade9f7e0bb91447bffc5e/49853.ipynb).

Could you please check in latest version and see if the issue still exists?

Thanks.
